docs/mesos.md

```diff
@@ -0,0 +1,47 @@
+# Mesos Support
+
+The MesosContainerFactory enables launching action containers within a Mesos cluster. It does not affect the deployment of OpenWhisk components (invoker, controller).
```

Comment:
perhaps backtick escape all the class names (maybe it would be nice to also link to the impl?)